src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -1,147 +1,175 @@
-import { useState } from 'react';
-import Form from 'react-bootstrap/Form';
+import React, { useState, useEffect } from 'react';
 import Button from '../../common/Button/Button';
-import Input from '../../common/Input/Input';
-import styles from '../CreateCourse/CreateCourse.module.css';
-import AuthorItem from './components/AuthorItem/AuthorItem';
+import { mockAuthors, mockCourses } from '../../data/data';
 import formatCreationDate from '../../helpers/formatCreationDate';
+import getCoursesDuration from '../../helpers/getCoursesDuration';
+import AuthorItem from './components/AuthorItem/AuthorItem';
+import Input from '../../common/Input/Input';
+import { useNavigate } from 'react-router-dom';
+import styles from './CreateCourse.module.css';
 import createGuid from '../../helpers/createGuid';
 
-const CreateCourse = ({ course, setCourses, author, setAuthor }) => {
-  let [inputs, setInputs] = useState({});
-  let [authorInput, setAuthorInput] = useState('');
-  let [formAuthors, setFormAuthors] = useState([]);
-
+function CreateCourse() {
+  const [authorInput, setAuthorInput] = useState('');
+  const [duration, setDuration] = useState(0);
+  const [courseAuthors, setCourseAuthors] = useState([]);
   let authorId = createGuid();
-  let courseId = createGuid();
-
-  const addAuthorToForm = (authorName, event) => {
-    event.stopPropagation();
-    if (formAuthors.includes(authorName)) return;
-
-    setFormAuthors([...formAuthors, authorName]);
-  };
-
-  const deleteAuthor = (authorName) => {
-    setFormAuthors(formAuthors.filter((author) => author !== authorName));
-  };
+  const navigate = useNavigate();
+
+  const [course, setCourse] = useState({
+    id: '0',
+    title: '',
+    description: '',
+    creationDate: '',
+    duration: 0,
+    authors: [authorId],
+  });
+
+  useEffect(() => {
+    if (course.title != '') {
```

Comment:
Please, always use [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) in JS to make sure you are using the same values with same types